src/resources/HorizontalPodAutoscalers/HorizontalPodAutoscalerDetails.js

```diff
@@ -99,14 +101,104 @@ export function HorizontalPodAutoscalerDetails(props) {
                   {t(m.i18label)} {m.name}
                 </>
               }
-              value={`${current[id] || EMPTY_TEXT_PLACEHOLDER} / ${m.value}`}
+              value={
+                current[id] === '1m'
+                  ? `0 / ${m.value}`
+                  : `${current[id] || EMPTY_TEXT_PLACEHOLDER} / ${m.value}`
+              }
             />
           ))}
         </LayoutPanel.Body>
       </LayoutPanel>
     );
   };
 
+  const HPABehavior = ({ spec }) => {
+    if (!spec?.behavior) return null;
+
+    const scaleUp = spec?.behavior?.scaleUp;
+    const scaleDown = spec?.behavior?.scaleDown;
+
+    const rowRenderer = policy => {
+      return [policy.type, policy.value, policy.periodSeconds];
+    };
+
+    return (
+      <LayoutPanel className="fd-margin--md " key="hpa-behavior">
+        <LayoutPanel.Header>
+          <LayoutPanel.Head title={t('hpas.headers.behavior')} />
+        </LayoutPanel.Header>
+        {scaleUp && (
+          <LayoutPanel className="fd-margin--md " key="hpa-behavior-scale-up">
+            <LayoutPanel.Header>
+              <LayoutPanel.Head title={t('hpas.headers.scaleUp')} />
+            </LayoutPanel.Header>
+            <LayoutPanel.Body>
+              <LayoutPanelRow
+                name={t('hpas.headers.stabilizationWindowSeconds')}
+                value={
+                  scaleUp?.stabilizationWindowSeconds ?? EMPTY_TEXT_PLACEHOLDER
+                }
+              />
+              <LayoutPanelRow
+                name={t('hpas.headers.selectPolicy')}
+                value={scaleUp?.selectPolicy ?? EMPTY_TEXT_PLACEHOLDER}
+              />
+            </LayoutPanel.Body>
+            {scaleUp?.policies && (
+              <GenericList
+                searchSettings={{ showSearchField: false }}
+                entries={scaleUp.policies}
+                key="behavior-scaleUp"
+                title={t('hpas.headers.scaleUp')}
+                headerRenderer={() => [
+                  t('hpas.headers.type'),
+                  t('hpas.headers.value'),
+                  t('hpas.headers.periodSeconds'),
+                ]}
+                rowRenderer={rowRenderer}
+              />
+            )}
+          </LayoutPanel>
+        )}
+        {scaleDown && (
+          <LayoutPanel className="fd-margin--md " key="hpa-behavior-scale-up">
+            <LayoutPanel.Header>
+              <LayoutPanel.Head title={t('hpas.headers.scaleDown')} />
+            </LayoutPanel.Header>
+            <LayoutPanel.Body>
+              <LayoutPanelRow
+                name={t('hpas.headers.stabilizationWindowSeconds')}
+                value={
+                  scaleDown?.stabilizationWindowSeconds ??
+                  EMPTY_TEXT_PLACEHOLDER
+                }
+              />
+              <LayoutPanelRow
+                name={t('hpas.headers.selectPolicy')}
+                value={scaleDown?.selectPolicy ?? EMPTY_TEXT_PLACEHOLDER}
```

Comment:
If you check whether `scaleDown` exists, there's no need to use optional chaining in '173 and '179. Error won't be thrown.